docs/codeql/conf.py

```diff
@@ -39,6 +39,10 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+# The default language for syntax highlighting, defaults to Python if omitted.
+
+highlight_language = "none"
```

Comment:
This will stop Sphinx from highlighting unlabeled code samples as Python!

Reply:
Update the comment above it!